comms/discovery/rpcz/chat_blast_test.go

```diff
@@ -161,11 +160,11 @@ func TestChatBlast(t *testing.T) {
 
 		blastCount := 0
 		for _, c := range chats {
-			if strings.HasPrefix(c.ChatID, "blast:") {
+			if c.IsBlast {
 				blastCount++
 			}
 		}
-		assert.Equal(t, "blast:69:follower_audience", chats[0].ChatID)
+		assert.Equal(t, "follower_audience", chats[1].ChatID)
```

Comment:
@stereosteve i'm not entirely sure why the order changed, i logged the chats here:
```
2024/08/21 22:35:46 INFO REED chats !BADKEY="[{ChatID:D79jn:eYZmn CreatedAt:2024-08-22 02:34:16.079029 +0000 UTC LastMessage:{String:what up fam Valid:true} LastMessageAt:2024-08-22 02:34:26.079029 +0000 UTC LastMessageIsPlaintext:true InviteCode:x LastActiveAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false} UnreadCount:1 ClearedHistoryAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false} IsBlast:false Audience:{String: Valid:false} AudienceContentType:{String: Valid:false} AudienceContentID:{Int32:0 Valid:false}} {ChatID:follower_audience CreatedAt:2024-08-22 02:34:26.079029 +0000 UTC LastMessage:{String:what up fam Valid:true} LastMessageAt:2024-08-22 02:34:26.079029 +0000 UTC LastMessageIsPlaintext:true InviteCode: LastActiveAt:{Time:2024-08-22 02:34:26.079029 +0000 UTC Valid:true} UnreadCount:0 ClearedHistoryAt:{Time:0001-01-01 00:00:00 +0000 UTC Valid:false} IsBlast:true Audience:{String:follower_audience Valid:true} AudienceContentType:{String: Valid:false} AudienceContentID:{Int32:0 Valid:false}}]"
```

Reply:
LastMessageAt are the same

Reply:
As reed and I discussed... it's because `getChat` query doesn't actually filter on `follower_audience`... so other test data is coming back when it shouldn't.  See steve todo below to fix this post-merge.